src/components/LocationWeather/LocationWeatherCurrent/LocationWeatherCurrent.jsx

```diff
@@ -0,0 +1,66 @@
+import {
+  API_SYMBOL_URL_POSTFIX,
+  API_SYMBOL_URL_PREFIX,
+  API_WIND_IMG_URL_POSTFIX,
+  API_WIND_IMG_URL_PREFIX,
+  WIND_DIRECTIONS_TO_IMG
+} from '../../../constants/constants';
+import { formatTemperature } from '../../../utils/utils';
+import './LocationWeatherCurrent.css';
+
+function LocationWeatherCurrent({ currentLocationWeather, currentLocationInfo }) {
+  const locationName = currentLocationInfo ? currentLocationInfo.name : null;
```

Comment:
Changed null to empty string.
I was just needed some value that React will not show. I did not know that empty string is better than null.
Please explain why it's better? - because of performance or because it visually more clear?

Reply:
Thanks